website/docs/guides/dynamic-loading-catalogs.md

```diff
@@ -1,12 +1,19 @@
+---
+title: Dynamic Loading of Message Catalogs
+description: Learn how to set up dynamic loading of message catalogs in Lingui to reduce bundle size and improve performance
+---
+
 # Dynamic Loading of Message Catalogs
 
-[`I18nProvider`](/docs/ref/react.md#i18nprovider) doesn't assume anything about your app and it's the developer responsibility to load messages based on active language.
+Internationalization in modern applications requires an efficient way to manage and load localized messages without overwhelming the initial bundle size. With Lingui's flexible approach, the developer is responsible for dynamically loading message catalogs based on the active language.
```

Comment:
I'd appreciate this paragraph to be more specific and open about what is (1) easy: load strings only for one active locale - which AFAIK is the default, but can still be quite wasteful (not 100% "efficient"), (2) possible: load messages on-demand using https://lingui.dev/ref/core#i18n.load
(3) hard / currently not possible but optimal: https://github.com/lingui/js-lingui/issues/1384


maybe also mention the experimental extractor?

Reply:
Thanks for the feedback, I just pushed the updated version.